go.mod

```diff
@@ -3,13 +3,46 @@ module github.com/mongodb/mongodb-kubernetes-operator
 go 1.13
 
 require (
-	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/hashicorp/go-multierror v1.0.0
+	github.com/operator-framework/operator-sdk v0.15.1
 	github.com/spf13/cast v1.3.1
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/zap v1.13.0
-	k8s.io/api v0.15.9
-	k8s.io/apimachinery v0.15.9
+	golang.org/x/net v0.0.0-20191028085509-fe3aa8a45271
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v12.0.0+incompatible
-	sigs.k8s.io/controller-runtime v0.3.0
+	sigs.k8s.io/controller-runtime v0.5.0
```

Comment:
is this intentional?

we should stick to 1.16 max
also client-go is now following k8s versioning